components/actionbar/index.css

```diff
@@ -10,50 +10,189 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-ActionBar {
-  --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
-  --spectrum-actionbar-padding-left: var(--spectrum-global-dimension-size-200);
-  --spectrum-actionbar-padding-right: calc(var(--spectrum-global-dimension-size-200) / 2);
-  --spectrum-actionbar-margin-x: var(--spectrum-global-dimension-size-200);
-  --spectrum-actionbar-offset-y: var(--spectrum-global-dimension-size-200);
-
-  --spectrum-actionbar-min-width: 280px;
-  --spectrum-actionbar-max-width: 960px;
+  --spectrum-actionbar-height: var(--spectrum-action-bar-height);
+  --spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-100);
+
+  /* item counter field label */
+  --spectrum-actionbar-item-counter-font-size : var(--spectrum-font-size-100);
```

Comment:
nit, remove spaces between ":"

Reply:
There were a few of those, thanks for this.